mne/preprocessing/ica.py

```diff
@@ -1367,7 +1377,8 @@ def _check_exclude(self, exclude):
         if exclude is None:
             return list(set(self.exclude))
         else:
-            return list(set(self.exclude + exclude))
+            return list(set(list(self.exclude) + list(exclude)))
```

Comment:
or `return list(set(self.exclude).union(set(exclude)))`?

Reply:
Right. That would save 2 empty spaces ;)

Reply:
No, it's more directly what you want to do right?

Reply:
The set() is around both lists, so it should have the same effect. But I have already changed it to your suggestion, because it looks nicer.